lib/matplotlib/axes/_axes.py

```diff
@@ -206,7 +206,7 @@ def set_xlabel(self, xlabel, fontdict=None, labelpad=None, **kwargs):
             The label text.
 
         labelpad : scalar, optional, default: None
-            Spacing in points between the label and the x-axis.
+            Spacing in points from the axes bounding box including ticks and tick labels.
```

Comment:
Could you make the lines <= 79 characters long by running the sentence on to the next line?

Reply:
@dstansby please check the latest commit (4). I have addressed the issue. Thank you!